### DISCUSSION TITLE
Reason for my rejection of GitHub action?

### DISCUSSION THREAD

**Discussion by jainal09** [community/community#26153] [Actions]
<p>Hi,<br>
I am a newbie in GitHub actions.<br>
Recently I created a GitHub actions as I had participated in the GitHub actions hackathon.<br>
But today i found out that my GitHub action was rejected! Stating “Not a suitable GitHub actions”</p>
<p>This is my GitHub action- <a href="https://github.com/jainal09/strivio" rel="nofollow noopener">https://github.com/jainal09/strivio</a></p>
<p>Its an awesome GitHub action were you can conduct a coding competetion like hackerrank/code-chef<br>
With functionalities like custom Input/Output test cases and Individual code reviews via CI on GitHub.</p>
<p>My question is What did I do wrong?<br>
Why was my action rejected?</p>
<p>Regards,<br>
Jainal</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p><a class="mention" href="/jainal09">@jainal09</a> For you to get response from Github Action Hackathon team directly, I would recommend you asking your question here: </p>
<p><a href="https://github.community/t/featured-event-github-actions-hackathon/13091">https://github.community/t5/Events/Featured-Event-GitHub-Actions-Hackathon/td-p/48206</a> Thank you for your understanding. </p>
